packages/settings-library/src/settings_library/prometheus.py

```diff
@@ -1,26 +1,35 @@
 from functools import cached_property
 
-from pydantic.networks import AnyHttpUrl
+from pydantic import AnyUrl, SecretStr
 from settings_library.base import BaseCustomSettings
 from settings_library.utils_service import MixinServiceSettings
 
-from .basic_types import PortInt, VersionTag
+from .basic_types import VersionTag
 
 
 class PrometheusSettings(BaseCustomSettings, MixinServiceSettings):
-    PROMETHEUS_HOST: str = "prometheus"
-    PROMETHEUS_PORT: PortInt = 9090
-    PROMETHEUS_VTAG: VersionTag = "v1"
+    PROMETHEUS_URL: AnyUrl
+    PROMETHEUS_VTAG: VersionTag = VersionTag("v1")
```

Comment:
If  you do not use host/port etc ... do you really need to inherit from MixinServiceSettings?

Reply:
I do not know. this is something that was already there and might be used in the web/server